File: src/tool/spring-boot-generator/output/contract/db/userConsumeRecord.js
let userConsumeRecord = {
	name: "userConsumeRecord",
	description: "",
	columns: {
		accountId: {
			name: "accountId",
			description: "账户编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		consumeAfter: {
			name: "consumeAfter",
			description: "消费后剩余",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		consumeAmount: {
			name: "consumeAmount",
			description: "消费份数",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		consumeBefore: {
			name: "consumeBefore",
			description: "消费前剩余",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		consumeType: {
			name: "consumeType",
			description: "消费类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		contractId: {
			name: "contractId",
			description: "合同编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		id: {
			name: "id",
			description: "编号",
			isPk: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		remark: {
			name: "remark",
			description: "备注",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		}
	}
};

let columnsArray = [
	"accountId",
	"consumeAfter",
	"consumeAmount",
	"consumeBefore",
	"consumeType",
	"contractId",
	"createTime",
	"id",
	"remark"
];

function create() {
  return {};
}

userConsumeRecord.columnsArray=columnsArray;
userConsumeRecord.create=create;
module.exports=userConsumeRecord;

File: src/tool/spring-boot-generator/output/contract/contract/src/main/java/com/contract/pojo/entity/UserConsumeRecord.java
package com.contract.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("")
@Getter
@Setter
public class UserConsumeRecord  {

    @ApiModelProperty("账户编号")
    private Integer accountId;

    @ApiModelProperty("消费后剩余")
    private Integer consumeAfter;

    @ApiModelProperty("消费份数")
    private Integer consumeAmount;

    @ApiModelProperty("消费前剩余")
    private Integer consumeBefore;

    @ApiModelProperty("消费类型")
    private Integer consumeType;

    @ApiModelProperty("合同编号")
    private Integer contractId;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("编号")
    private Integer id;

    @ApiModelProperty("备注")
    private String remark;

}
